product-bundles-variation-bundles.php

```diff
@@ -122,7 +122,7 @@ public static function load_plugin() {
 		add_filter( 'woocommerce_product_variation_get_price', array( __CLASS__, 'variation_get_price' ), -1000, 2 );
 		add_filter( 'woocommerce_product_variation_get_regular_price', array( __CLASS__, 'variation_get_regular_price' ), -1000, 2 );
 		add_filter( 'woocommerce_product_variation_get_sale_price', array( __CLASS__, 'variation_get_sale_price' ), -1000, 2 );
-		add_filter( 'woocommerce_variation_prices', array( __CLASS__, 'variation_prices' ), -1000, 2 );
+		add_filter( 'woocommerce_variation_prices', array( __CLASS__, 'variation_prices' ), -1000, 3 );
```

Comment:
Was this in error?

Reply:
Yes and no. This hook requires 3 arguments: https://github.com/woocommerce/woocommerce/blob/3611d4643791bad87a0d3e6e73e031bb80447417/plugins/woocommerce/includes/data-stores/class-wc-product-variable-data-store-cpt.php#L382 but the 3rd one is optional.